### PR TITLE
Rename env variable from `DATA_PLANE_CONFIG_MAP` to `CONTRACT_CONFIG_MAP`

### PR DESCRIPTION
**Pull request by pierDipi**
The previous name was too generic and wasn't clear on the actual subject.

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

Part of https://github.com/knative-sandbox/eventing-kafka-broker/pull/2601#discussion_r962978823

### CODE REVIEW

**Comment by pierDipi** [comment]
Transfer failed:
/test unit-tests_eventing-kafka-broker_main

File: control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml [hunk 1]
```diff
@@ -69,24 +69,24 @@ spec:
             - name: SOURCE_DATA_PLANE_CONFIG_MAP_NAMESPACE
               value: knative-eventing
 
-            - name: BROKER_DATA_PLANE_CONFIG_MAP_NAME
+            - name: BROKER_CONTRACT_CONFIG_MAP_NAME
```
Comment (matzew) [inline control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml:72]: I like that we rename them separately as a step before touching this section of code

**Comment by matzew** [comment]
/retest

seeing some `timeout` errors

**Comment by aliok** [comment]
/hold
/lgtm
/approve

Thanks for the renaming, I love it.

Just added a hold because of the failing `gofmt` issue. Feel free to unhold if it is not a real issue.

**Comment by pierDipi** [comment]
@aliok thanks, done!

**Review by aliok — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**